### ISSUE TITLE
FF38 just installed a new update and the toolbar icon disappeared

### ISSUE THREAD

**Issue by DisconsolateChimera** [RequestPolicy/requestpolicy#457]
Please forgive me because I really don't know how to use GitHub. I got here after the upgrade to FF38 broke RP and managed to fix that. Now Mozilla has sent an update to 38 and the toolbar icon, our little red flag, has disappeared. I have no idea how to get it back. I'd appreciate any help.

**Comment by myrdd** [comment]
Hi @DisconsolateChimera welcome to github!

Please go to `about:addons` to see if RequestPolicy is still installed. It might just got disabled. If it's disabled, click its „enable“ button.
In case RP disappeared completely, try to install it again: https://addons.mozilla.org/en-US/firefox/addon/requestpolicy/

Please tell me whether it worked.

**Comment by DisconsolateChimera** [comment]
I had already done that, and RequestPolicy is still installed and enabled. Funny thing, though. The icon has now, today, suddenly reappeared. ??? I'm not quite sure what to think. But it is back now.

**Comment by jhjensen** [comment]
Firefox is telling me that request policy is disabled because it it not compatible with 38.0.1.  If I try and install it, the page won't let me do that.

**Comment by myrdd** [comment]
I have just forked the old _0.5_ branch and created a new addon on AMO. It's still awaiting review, but you can install and test it. See https://addons.mozilla.org/en-US/firefox/addon/requestpolicy-continued/

**Comment by DisconsolateChimera** [comment]
Thank you. I did so. And, it fixed my again-disappeared icon! Yes, today when I opened FF the icon was gone. I came here to talk to you about it and saw your note so went and installed your addon. Afterwards, after restarting FF the icon is back again.

I'm wondering if it's having some kind of conflict with some other addon's icon's placement. I keep the flag in the bottom right of the add-on tool bar, not up in the top right where your program puts it. I don't really see how that could make a difference but that's the only thing I can think of. I do have quite a few security adds down there with it!

**Comment by DisconsolateChimera** [comment]
@jhjensen When that happened to me, I went here: https://www.requestpolicy.com/1.0.html Go just about 1/2-way down the screen to "Install the RequestPolicy 1.0 beta." Do that. The beta installed just fine for me. Then when my icon started flickering on and off @myrdd kindly started holding my hand to try and solve that issue.

**Comment by myrdd** [comment]
@DisconsolateChimera I'm glad this works now.

> I went here: https://www.requestpolicy.com/1.0.html Go just about 1/2-way down the screen to "Install the RequestPolicy 1.0 beta."

If you want to use the 1.0 beta, please install it from https://requestpolicycontinued.github.io/. The difference is that it's the _maintained_ version by the [„RequestPolicyContinued“ project](https://github.com/RequestPolicyContinued/requestpolicy).

**Comment by Lumikai** [comment]
@myrdd Just to be sure, is the forked 0.5 version you mention here just a FF38 compatibility update for the original requestpolicy?:
https://addons.mozilla.org/en-US/firefox/addon/requestpolicy-continued/

RequestPolicy is one of my favourite firefox addons but I'm not quite ready to move on to RPC 1+ beta =). I'm currently running firefox 37.0.2 while I attempt to get the original RequestPolicy working for FF38.
Many thanks.

**Comment by myrdd** [comment]
> is the forked 0.5 version you mention here just a FF38 compatibility update for the original requestpolicy

Precisely. There have been some changes, see the [0.5.29 release notes](https://github.com/RequestPolicyContinued/requestpolicy/releases/tag/v0.5.29) and the [diff with 0.5.28](https://github.com/RequestPolicyContinued/requestpolicy/compare/release-0.5.28...v0.5.29) for details.

With RPContinued 0.5.29 you shouldn't have any problems on FF38.

**Comment by DisconsolateChimera** [comment]
@Lumikai: I'm the original poster of this question. I installed 0.5.29 
after installing the /proper/ RP to my FF38 (i.e., from 
https://requestpolicycontinued.github.io/ thanks Martin!) and now it's 
all running fine. I really didn't want to undo the update to FF which 
happened automatically. (Changed that back to not!)

Speaking of update to 38, just FYI and unrelated to this, if you'd like 
to know, they have moved Options from being a popup to opening in a new 
tab. Hate it! Of course, I haven't got around yet to going to forums to 
see how to undo that, but still... :/

Jean

On 5/27/2015 10:57 AM, Lumikai wrote:

> @myrdd https://github.com/myrdd Just to be sure, is the forked 0.5 
> version you mention here just a FF38 compatibility update for the 
> original requestpolicy?:
> https://addons.mozilla.org/en-US/firefox/addon/requestpolicy-continued/
> 
> RequestPolicy is one of my favourite firefox addons but I'm not quite 
> ready to move on to RPC 1+ beta =). I'm currently running firefox 
> 37.0.2 while I attempt to get the original RequestPolicy working for FF38.
> Many thanks.
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/RequestPolicy/requestpolicy/issues/457#issuecomment-105974112.

---

This email has been checked for viruses by Avast antivirus software.
http://www.avast.com

**Comment by myrdd** [comment]
> they have moved Options from being a popup to opening in a new tab.

See `browser.preferences.inContent` / http://techdows.com/2015/04/how-to-disable-tab-based-preferences-in-firefox-38.html

**Comment by Lumikai** [comment]
@DisconsolateChimera Thanks Jean, thats really good to hear (RP 0.5.29 working, not the new firefox options tab)

@myrdd Just wanted to express my gratitude for your continued work on RequestPolicy; especially in keeping the original alive while you work on the new version. Thank you.

**Comment by DisconsolateChimera** [comment]
YES! _ahem_ Thanks. Saved my lazy self from going looking for that.

On 5/29/2015 2:05 AM, Martin Kimmerle wrote:

> ```
> they have moved Options from being a popup to opening in a new tab.
> ```
> 
> See |browser.preferences.inContent| / 
> http://techdows.com/2015/04/how-to-disable-tab-based-preferences-in-firefox-38.html
> 
> —
> Reply to this email directly or view it on GitHub 
> https://github.com/RequestPolicy/requestpolicy/issues/457#issuecomment-106716379.

---

This email has been checked for viruses by Avast antivirus software.
http://www.avast.com
